toolset/databases/db4j/db4j.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM maven:3.5.3-jdk-11-slim as maven
+WORKDIR /db4j
+COPY pom.xml pom.xml
+RUN mvn dependency:copy-dependencies -DoutputDirectory=libs
+COPY src src
+RUN mvn install -q
+
+FROM openjdk:11-jre-slim
+WORKDIR /db4j
+COPY --from=maven /db4j/libs libs
+COPY --from=maven /db4j/target target
+CMD java -cp target/classes:libs/\* hello.HelloData true
+
+
+
```

Comment:
It's unlikely we're going to add this database to the toolset. https://github.com/db4j/db4j

Reply:
thanks for taking a look at my code

what is the criteria for inclusion ? i inquired previously and was led to believe that the only requirements were that databases be client-server and persist data to disk

https://github.com/TechEmpower/FrameworkBenchmarks/issues/2209